pkg/invariantlibrary/allowedbackenddisruption/matches.go

```diff
@@ -21,10 +19,5 @@ const (
 // GetAllowedDisruption uses the backend and information about the cluster to choose the best historical p95 to operate against.
 // We enforce "don't get worse" for disruption by watching the aggregate data in CI over many runs.
 func GetAllowedDisruption(backendName string, jobType platformidentification.JobType) (*time.Duration, string, error) {
-	// Special case for the liveness probe to external service:
-	if backendName == externalservice.LivenessProbeBackend {
-		aed := allowedExternalDisruption
-		return &aed, "forgiving limit for disruption to an external service", nil
-	}
```

Comment:
IIRC This was in place so we didn't fail due to our check to see if the cluster running the tests was having connection issues.

Reply:
The test was always ok. I've left the backend check and it reports intervals and overall disruption. Do we also need the "never fail" test?